modules/chempy/cpv.py

```diff
@@ -403,5 +393,3 @@ def fit(target_array, source_array):
             return(t1, t2, rot_mtx, rmsd)
 
 # Too many iterations; something wrong.
-    print ("Error: Too many iterations in RMS fit.")
-    raise ValueError
```

Comment:
Why remove this exception? Given that the rest of this refactoring only adds type annotations, this is an unexpected behavioral change. I would expect at least that such a change is discussed in the pull request description. I would also add to the docstring under which conditions (e.g. no convergence) the function returns `None`.